it_hardware/__manifest__.py

```diff
@@ -0,0 +1,70 @@
+{
+    'name': 'IT Hardware Sales and Support',
+    'version': '1.0',
+    'category': 'Services',
+    'description': """
+We offer IT sales, installation, and repair services for a wide range of products, including laptops, RAM, accessories, and CCTV cameras.
+""",
+    'depends': [
+        'account_followup',
+        'helpdesk_stock',
+        'knowledge',
+        'product_expiry',
+        'purchase_stock',
+        'industry_fsm_sale',
+        'repair',
```

Comment:
Why do we need industry_fsm? I thought from the knowledge that the whole logic was handled with repair, isn't it?
```suggestion
        'repair',
```

Reply:
<field name="is_fsm" eval="True"/> 
As we have this field so we need the industry_fsm dependency